extension/lib/doc.js

```diff
@@ -106,6 +106,13 @@ class Doc {
                 let start = sentenceBoundary.index;
                 let end = start + sentenceBoundary.offset;
                 this.sentences.push(new SentencePointer(container_id, start, end));
+                let sentenceId = this.sentences.length - 1;
```

Comment:
Very nice approach man :)))))
Love it. Makes this CL simple.
-- Feel free to resolve.

Reply:
Thanks!